docs/features/contributor-insights.md

```diff
@@ -47,7 +47,7 @@ On your Contributor Insight Page, you'll see that each contributor is assigned a
 The OSCR takes into account various factors related to all of the contributor's activity and the quality of their contributions in open source. 
 
 :::info
-OSCRs are calculated on a rolling 30-day basis across all of a contributor's activity. This means the score reflects recent activity and engagement, providing a current snapshot of a contributor's open source involvement rather than a lifetime aggregate.
+OSCRs are calculated on a rolling 30-day basis across all of a contributor's activity and are out of 300 total points. This means the score reflects recent activity and engagement, providing a current snapshot of a contributor's open source involvement rather than a lifetime aggregate.
```

Comment:
This actually reminded me of something that I forgot had been implemented: to give the OSCR a broader breadth and depth, the range was expanded to 90 days. That way, when someone is on vacation or taking a break, their score won't immediately tank.

```suggestion
OSCRs are calculated on a rolling 90-day basis across all of a contributor's activity and are out of 300 total points. This means the score reflects recent activity and engagement, providing a current snapshot of a contributor's open source involvement rather than a lifetime aggregate.
```

appologies: I'd meant to get in the docs and update them after adding that but got side tracked 🤠 😅